.github/workflows/release.yml

```diff
@@ -278,19 +278,55 @@ jobs:
           packages_dir: wheels/
           repository_url: https://test.pypi.org/legacy/
 
-  build-docs:
+  create-binder-env:
     needs: [deploy]
     runs-on: ubuntu-latest
+    steps:
+      - name: Get scikit-decide release version and define new binder env tag
+        run: |
+          TAG_NAME=${GITHUB_REF/refs\/tags\//}  # stripping "refs/tags/"
+          SKDECIDE_VERSION=${TAG_NAME/v/}  # stripping "v"
+          BINDER_TAG=binder_${TAG_NAME}
+          echo "BINDER_TAG=${BINDER_TAG}" >> $GITHUB_ENV
+          echo "SKDECIDE_VERSION=${SKDECIDE_VERSION}" >> $GITHUB_ENV
+
+      # Checks-out binder branch
+      - name: checkout binder branch
+        uses: actions/checkout@v2
+        with:
+          ref: binder
+
+      - name: Update scikit-decide dependency and tag the environement
+        run: |
+          sed -e "s/scikit-decide\[all\]==.*/scikit-decide\[all\]==${SKDECIDE_VERSION}/" environment.yml > environment.yml.new
+          mv environment.yml.new environment.yml
+          git config user.name "Binder Bot"
+          git config user.email "binder@bot.com"
+          git commit environment.yml -m "Update scikit-decide dependency"
+          git tag -a ${BINDER_TAG} -m "Release binder environment for scikit-decide ${SKDECIDE_VERSION}"
+          git push origin ${BINDER_TAG}
+
+      - name: Cache binder build on mybinder.org
```

Comment:
Why do we want to create this cache?

Reply:
To avoid a long waiting time for the first user who will click on the links in the doc. It is not a big deal that the release workflow takes a bit more time imho (as it is for release and not each push)

Reply:
I will take care of binder cache in an other PR. The way it is done in repo2docker-action is perfectible and it will complexify this PR for nothing